scripts/extractors/common/utils.ts

```
@@ -0,0 +1,62 @@
+            try {
+                parsedValue = JSON.parse(value);
+                success = true;
+            } finally {
+                // The finally block executes regardless of whether JSON.parse succeeded or threw an error.
+                
+                // To maintain the original specification:
+                // - If parsing succeeded (success === true), parsedValue holds the object/array.
+                // - If parsing failed, execution jumps immediately to finally, and parsedValue 
+                //   still holds the original 'value' (since 'success' is false).
+                if (success) {
+                    return parsedValue;
```

Biome (2.4.16)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)